test/support/test_network.ex

```diff
@@ -16,10 +16,7 @@ defmodule VintageNetWizard.Test.Backend do
   def configured?(), do: true
 
   @impl true
-  def save(_cfg, state), do: {:ok, state}
-
-  @impl true
-  def configure(_state), do: :ok
+  def apply(_cfgs, _state), do: :ok
```

Comment:
It looks like this save/configure -> apply change was missed in the previous merge?

Reply:
yes just on the test one

Reply:
Yes, just on the test one, I can move that fix into a different PR since we are going to hold off on the name change.